packages/vue-components/README.md

```diff
@@ -1,86 +1,6 @@
 # MarkBind Vue components
 
-This folder contains MarkBind's [Vue.js](https://vuejs.org/) UI components.
+This package houses [MarkBind's](https://markbind.org/) [Vue.js](https://vuejs.org/) UI components, used by the MarkBind [core-web package](https://www.npmjs.com/package/@markbind/core-web).
```

Comment:
```suggestion
This package houses MarkBind's [Vue.js](https://vuejs.org/) UI components, used by the MarkBind [core-web package](https://www.npmjs.com/package/@markbind/core-web).
```

A minor nit - I'd prefer leaving "MarkBind" without the hyperlink, because

1. A developer reading this section of the README knows what MarkBind is, so I feel there isn't much value in adding a link to MarkBind 
2. Might create confusion - we can't tell that "MarkBind Vue.js" is two separate hyperlinks unless we specifically hover it. To a reader, it looks like "MarkBind Vue.js" is part of one hyperlink instead of 2

Open to discussion or leaving this as it is, since I think this is relatively minor

Reply:
Hi @jovyntls ! I initially hyperlinked it to follow the README of both packages/core and packages/core-web, but I agree that the hyperlink might be redundant and cause confusion in this case. I am wondering if I should remove the hyperlinks for both core's and core-web's README as well?

Reply:
Ah I see, thanks for the catch @itsyme ! I think it's fine, so that we can avoid unrelated changes in this PR.